### ISSUE TITLE
Get rid of html font-size: 62.5%

### ISSUE THREAD

**Issue by benhinchley** [whatsnewsaes/Boneless#11]
In reference to this issue [#18](http://github.com/whatsnewsaes/Skeleton-Sass/issues/18). 

I think we should defiantly get rid of the 62.5% _trick_ and move to em/rem based type system

**Comment by WhatsNewSaes** [comment]
Absolutely.

I tend to favor em's because they have better browser support. However, I'm down to use either.

**Comment by ckihneman** [comment]
I used to use skeleton years ago, for a long time, as my go to for any frontend work. I still live in a multi-page website world that needs to support IE8+. REMs are "fancy" and all, but at the end of the day they are just that, something fancy web devs can play with, that only serve to alienate people like myself. We have tried and true methods that are backwards compatible, I humbly suggest you stick to `em` and `px` values (which it looks like you have).

I applaud the work you guys are doing here, for sure the direction I wish skeleton had gone years ago. I'll be looking over this work more and see if its something I can get into further. Thanks!
